### ISSUE TITLE
implicit defaultFormBinding not found after migrating from Play 2.8.5 to 2.8.6

### ISSUE THREAD

**Issue by mverbist** [playframework/playframework#10881]
### Play Version

2.8.6

### API

Scala 2.13.4

### Operating System
```
Linux 5.4.0-73-generic #82~18.04.1-Ubuntu SMP Fri Apr 16 15:10:02 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

### JDK

```
openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.18.04)
OpenJDK 64-Bit Server VM (build 11.0.11+9-Ubuntu-0ubuntu2.18.04, mixed mode, sharing)
```

### Expected Behavior

1. expect to find the `implicit val defaultFormBinding` from `BaseControllerHelpers` in statement `myForm.bindFromRequest()(request)`

### Actual Behavior

1. not enough arguments for method bindFromRequest: (implicit request: play.api.mvc.Request[_], formBinding: play.api.data.FormBinding): play.api.data.Form[models.view.Event].
Unspecified value parameter formBinding.
2. In fact, when I replace the failing code with `myForm.bindFromRequest()(request, defaultFormBinding)` it works, but I think it should find it implicitly. When I click through on the `defaultFormBinding` in IntelliJ, it even arrives on the correct defaultFormBinding

**Comment by mkurz** [comment]
Does that happen with latest 2.8.8 as well?

**Comment by mverbist** [comment]
I wasn't able to try with 2.8.8, because there I had other issues
But I'll give it an other try. And see if I can fix those other issues, if this issue still persists.
I will keep you posted.

**Comment by mverbist** [comment]
I tried it with Play 2.8.8, same issue occurs

**Comment by tsuyoshizawa** [comment]
This commit has broken compatibility.
https://github.com/playframework/playframework/commit/f453ddabb196966c74e743d27cc5765716f6e73f

It has been released in 2.8.6.

My project also had a compile error after upgrade the version.
I modified the project code to fix the compilation error.

I think that compatibility should have been maintained.

**Comment by mkurz** [comment]
> I think that compatibility should have been maintained.

Yes you are right, that should not have happened.

Nevertheless I am closing this issue because 2.8.6 was released long time ago and there is nothing we will/can do about this now.
